packages/manager/modules/pci/src/components/project/regions-list/regions-list.controller.js

```diff
@@ -75,6 +72,16 @@ export default class RegionsListController {
     }
   }
 
+  static isRegionDisabled(region) {
+    return region.length === 1 && !region[0].hasEnoughQuota;
+  }
+
+  onMacroChange(macro, regions) {
+    if (regions.length === 1) {
+      this.onRegionChange(regions[0]);
```

Comment:
```suggestion
      this.onRegionChange(first(regions));
```
not necessarily better too